hardik/hangman/main.rb

```diff
@@ -0,0 +1,38 @@
+puts 'Welcome to the Hangman Game'
+puts '-----------------------------------------------------------------'
+
+# Constants
+# --------------------------------------------
+WORDS = %w[elephant computer switch kylas ball flower okay prodigy enigma bottle].freeze
+TARGET = WORDS.sample.upcase
+LENGTH = TARGET.length
+# --------------------------------------------
+
+lives = 5
+user_guess = ['-'] * LENGTH
+# --------------------------------------------
+
+def show(user_guess, lives)
+  user_guess.each{ |c| print(" #{c} ") }
+  puts "\t\tLives #{lives}"
+end
+
+while lives >= 0
+  show(user_guess, lives)
+  unless user_guess.include?('-')
+    puts "\nCongratulations you won"
```

Comment:
You can do it like this. The readability of message is intact here.
> puts
puts "Congratulations! You Won."